nautobot/extras/choices.py

```diff
@@ -488,13 +488,17 @@ class SecretsGroupSecretTypeChoices(ChoiceSet):
     TYPE_SECRET = "secret"  # noqa: S105  # hardcoded-password-string -- false positive
     TYPE_TOKEN = "token"  # noqa: S105  # hardcoded-password-string -- false positive
     TYPE_USERNAME = "username"
+    TYPE_URL = "url"
+    TYPE_NOTES = "notes"
 
     CHOICES = (
         (TYPE_KEY, "Key"),
         (TYPE_PASSWORD, "Password"),
         (TYPE_SECRET, "Secret"),
         (TYPE_TOKEN, "Token"),
         (TYPE_USERNAME, "Username"),
+        (TYPE_URL, "URL"),
+        (TYPE_NOTES, "Notes"),
```

Comment:
Would `Note` be more accurate than `Notes`? I'm lacking some context here.

Reply:
Hi @glennmatthews 

This is the field name of the Delinea Secret Server that the API returns when reading a secret. There is also the Notes result Type as in https://github.com/nautobot/nautobot-app-secrets-providers choices.py

Reply:
@glennmatthews 
I'm using the URL defined at a particular Secret to reference the server I'm contacting and use the username and password that is also stored in SecretServer. The Notes field I use to define additional context like (is_production_server, ...) etc. The app uses the Notes field to parse the contents of the Notes field